README.rst

```diff
@@ -1,4 +1,4 @@
 sphinx-es-suggest
```

Comment:
Just a note but the name here doesn't match the name of the repository. Should we settle on a name? I think Sphinx Suggest is pretty good. Maybe the repository could be called "readthedocs-sphinx-suggest" and "readthedocs-" is just a prefix.

Reply:
Folders are also not matching with the name currently.
I am thining to update these things when the PR gets approved and is ready to merge.
@ericholscher suggested the name `readthedocs-sphinx-search`. If everyone doesn't agrees on this one, I think we can change it anytime (before uploading the package to pypi). Right?